systers_portal/community/forms.py

```diff
@@ -9,7 +9,30 @@
 from users.models import SystersUser
 
 
-class CommunityForm(ModelFormWithHelper):
+class AddCommunityForm(ModelFormWithHelper):
+    """ Form to create a new Community by admin. """
+    class Meta:
+        model = Community
+        fields = ('name', 'slug', 'order', 'email', 'mailing_list',
+                  'parent_community', 'website', 'facebook', 'googleplus',
+                  'twitter')
```

Comment:
First four should be required. [~slug, order]

Reply:
name, slug, order, admin are required fields currently.  Should I also make 'order' a required field?